Userland/Utilities/less.cpp

```diff
@@ -214,8 +271,11 @@ static String get_key_sequence()
 int main(int argc, char** argv)
 {
     const char* filename = "-";
+    const char* prompt = "?f%f :.(line %l)?e (END):.";
```

Comment:
It'd be nice to have a manpage for this, and it'd be even nicer if that manpage explains this format (or rather, what we support of it)

Reply:
Also, `char const*`.